### DISCUSSION TITLE
Remove top clock from titlebar on login screen

### DISCUSSION THREAD

**Discussion by egor-khanko** [pop-os/pop#2559] [Ideas]
Hey everyone! What do you think about removing clock and date from titlebar (hope I'm naming it correctly :sweat_smile:) from login screen, cause we have this info already from central clock/date.
I think this would make login screen feel cleaner)
This is what I'm talking about (both red boxes have same info)
![image](https://user-images.githubusercontent.com/32592704/182937801-75712636-69cd-4724-88d4-759ba9a999aa.png)
Sorry for bad quality, had to take photo of the screen instead of screenshot)

**Comment by jacobgkau** [comment]
This UX suggestion makes sense to me on the surface. It would create a situation after you start typing a password where the clock will not be visible (unless the top bar clock un-hides when the center-of-screen clock goes away.)

**Comment by jacobgkau** [comment]
The titlebar is no longer visible with cosmic-greeter in the new COSMIC Epoch. This will therefore be implemented in Pop!_OS 24.04 when COSMIC Epoch is released.
